src/options/optionsMain.js

```diff
@@ -33,6 +33,8 @@ export default function (inpOptions) {
     minHeight: inpOptions.minHeight,
     maxWidth: inpOptions.maxWidth,
     maxHeight: inpOptions.maxHeight,
+    x: inpOptions.x,
```

Comment:
@mdh34 thanks for the surprise-PR 🙂! So, what happens if I don't pass `--x`/`--y` here? x gets `undefined`, right? Have you confirmed the `new BrowserWindow({` later call in `mainWindow.js` is later on happy with that?

Reply:
@ronjouch upon testing the behaviour is normal without any arguments - no console errors either :)

Reply:
@ronjouch setting a default might not work that well either because of different resolution screens?

Reply:
@mdh34 yeah, I was not thinking of setting a default, I was thinking of only adding `x`/`y` to `inpOptions` when we have them from `inpOptions`. Else let's not set it, it's more explicit that we're letting Electron BrowserWindow do its thing with default values:

```javascript
if (inpOptions.x) {options.x = inpOptions.x;}
if (inpOptions.y) {options.y = inpOptions.y;}
```

Reply:
Ah yes sounds like a good idea :)